### ISSUE TITLE
Grind House skate shop shows blank

### ISSUE THREAD

**Issue by Kita2345** [scoute-dich/browser#661]
Blank page when visiting the following web site:
https://www.grindhouse.eu/

Sounds like an issue with WebView as it works on Chrome and does not with DuckDuckGo. Almost looks like the redirect is not working. Works on GeckoView with Firefox Focus.

**Comment by tusch001** [comment]
The reason may be an invalid certificate. In another app I got the message
```
Sorry, the SSL certificate seemed not to be trusted.
For security reason the provided url could not be fetched. 

java.security.cert.CertPathValidatorException: 
Trust anchor for certification path not found.
```
http not working, because rewrite mod to https active? (This is good)  
I looked at the certificates. Nothing obvious wrong. Only that in the cert chain the intermediate certificate doesn't end with `=`. Not sure, but I thought all certificates would end with that character. Would explain the above message about missing trust.

Anyway: should the browser alert about that?
